client/src/screens/UnconfiguredScreen.tsx

```diff
@@ -64,6 +65,17 @@ const UnconfiguredScreen = () => {
     history.push(routerPaths.electionDefinition)
   }
 
+  const saveElectionAndShowSuccess = useCallback(
+    (election: Election) => {
+      setShowSuccess(true)
+      setTimeout(() => {
+        saveElection(election)
+        setShowSuccess(true)
```

Comment:
False?

I'd also move this line above the saveElection line to ensure that the update happens before the component is unmounted. 

Or better yet, delete this line? Does the component state persist after election config is removed? I'm not certain, should test.

Reply:
good catch, going to keep it in there *just* to be extra pedantic about keeping the state clean, in case this component gets reused in the future. And will move to before the `saveElection`.